frontend/src/lib/components/AnnotationsOverlay/annotationsOverlayLogic.test.ts

```diff
@@ -668,12 +713,13 @@ describe('annotationsOverlayLogic', () => {
                     { value: 0 },
                     { value: 2 }, // This indicates that the ratio of ticks to points is 1:2
                 ],
+                dashboardId: MOCK_DASHBOARD_ID,
             })
             logic.mount()
             await expectLogic(annotationsModel).toDispatchActions(['loadAnnotationsSuccess'])
-            await expectLogic(insightLogic({ dashboardItemId: MOCK_INSIGHT_SHORT_ID })).toDispatchActions([
-                'loadInsightSuccess',
-            ])
+            await expectLogic(
+                insightLogic({ dashboardItemId: MOCK_INSIGHT_SHORT_ID, dashboardId: MOCK_DASHBOARD_ID })
```

Comment:
Worth adding a couple of test cases without `dashboardId` _and_ with an irrelevant one

Reply:
Did [here](https://github.com/PostHog/posthog/pull/23556/commits/9eeca37ddef158907113f506fcaf8e9bf9ac3e05) :)